docs-src/guide-rpc.scrbl

```diff
@@ -0,0 +1,59 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag     "guide-rpc"
+  #:style   'toc
+  ]{The Reach RPC interface}
+
+@(define link-client-go  @reachexlink["tut-6-rpc/client-go/index.go" "Go"])
+@(define link-client-py  @reachexlink["tut-6-rpc/client-py/index.py" "Python"])
+@(define link-react-code @reachexlink["overview-react" "overview-react example"])
+
+@(define link-rpc
+  @link["https://en.wikipedia.org/wiki/Remote_procedure_call"]{remote procedure call})
+
+@(define link-react-tube
+  @link["https://youtu.be/jHEKIMNvs-o"]{7-minute YouTube video})
+
+@DApps implemented in Reach are comprised of multiple interrelated but
+nonetheless fully-separate components. Of these, a given @DApp's
```

Comment:
We try to use "semantic linefeeds" (https://rhodesmill.org/brandon/2012/one-sentence-per-line/) meaning at least one newline per sentence, and sometimes additional newlines for some commas and semicolons